doc/source/index.rst

```diff
@@ -69,3 +69,27 @@ other control systems.
    :caption: API
 
    api
+
+.. toctree::
+   :hidden:
+   :caption: Data Collection
+
+   bluesky <https://nsls-ii.github.io/bluesky>
+   ophyd <https://nsls-ii.github.io/ophyd>
+
+.. toctree::
+   :hidden:
+   :caption: Data Access and Management
+
+   databroker <https://nsls-ii.github.io/databroker>
+   amostra <https://nsls-ii.github.io/amostra>
```

Comment:
As amostra and datamuxer and both unmaintained, I think we should not advertise them so prominently (for now). Would you mind removing them?